src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -94,7 +94,7 @@ export class AddBulkConcepts extends Component {
           </div>
         </fieldset>
         <h4>Concept IDs to add</h4>
-        <div className="prefered-concepts">
+        <div className="preffered-concepts">
```

Comment:
You just changed from `prefered` to `preffered` and below you change from `preffered` to `preferred`. Which is which. I think unless the spelling issue is the actual bug. I suggest you use `preffered` as its already consistent and then raise another ticket to change it to `preferred`. How is that?

Reply:
I will raise a ticket to rename the `preffered`s that not causing the bug.

Reply:
Here is the ticket, https://issues.openmrs.org/browse/OCLOMRS-272

Reply:
If you think the ticket is properly described for a new comer to start working on it immediately, can you make it ready for work?

Reply:
Yes, I can. I have done that.